rust/flatbuffers/src/builder.rs

```diff
@@ -699,13 +777,13 @@ impl<'fbb> FlatBufferBuilder<'fbb> {
         );
 
         while self.unused_ready_space() < want {
-            self.grow_owned_buf();
+            self.grow_allocator();
         }
         want
     }
     #[inline]
     fn unused_ready_space(&self) -> usize {
-        self.head
+        self.allocator.len() - self.head.range_to_end().count()
```

Comment:
optional nit: `.range_to_end().count()` is a little awkward, maybe use a single (e.g. `.distance_to_end()`) method?

Reply:
Done.